### PR TITLE
BL-2653 Get Ctrl+N to work on reader custom pages

### PR DESCRIPTION
**Pull request by gmartin7**

### CODE REVIEW

File: src/BloomExe/Edit/EditingModel.cs [hunk 1]
```diff
@@ -671,13 +671,13 @@ public void HandleAddNewPageKeystroke(string unused)
 				if (CanDuplicatePage)
 				{
 					AddNewPageBasedOnTemplate(this._pageSelection.CurrentSelection.IdOfFirstAncestor);
-					return;
-				}
-				else
-				{
-					var idOfFirstPageInTemplateBook = CurrentBook.FindTemplateBook().GetPageByIndex(0).Id;
-					AddNewPageBasedOnTemplate(idOfFirstPageInTemplateBook);
+					if (_addNewSuccess)
```
Comment (hatton) [inline src/BloomExe/Edit/EditingModel.cs]: This doesn't warrant introducing a new member variable. Just send a return value and act on it accordingly.

Comment (gmartin7) [inline src/BloomExe/Edit/EditingModel.cs]: Can't. See comment on line 708.

Comment (hatton) [inline src/BloomExe/Edit/EditingModel.cs]: Just wrap your call to AddMessageEventListener:

_view.AddMessageEventListener("addPage", (id) => AddNewPageBasedOnTemplate(id));

Comment (gmartin7) [inline src/BloomExe/Edit/EditingModel.cs]: done. Thanks a bunch. I didn't know you could do that!

**Pull request merged**